charts/clamav/values.yaml

```diff
@@ -0,0 +1,71 @@
+# Default values for clamav.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+nameOverride: ""
+fullnameOverride: ""
+
+servicePort: 443
+
+serviceAccount:
+  annotations: {}
+  name: clamav-notify
+
+clamavNotify:
+  image: quay.io/ukhomeofficedigital/acp-clamav-notify
+  version: ""
+  freshclam:
+    mirrors:
+      - db.uk.clamav.net
+      - database.clamav.net
+  volume:
+    storageClass: gp2-encrypted
+    size: 5
+
+clamavHTTP:
+  image: quay.io/ukhomeofficedigital/acp-clamav-http
+  version: ""
+  resources:
+    limits:
+      cpu: 400m
+      memory: 3000M
+
+clamav:
+  image: quay.io/ukhomeofficedigital/acp-clamav
+  version: ""
```

Comment:
For all the `version: ""` in this file, would it be better / more useable to comment those lines out and add comments describing what the default versions are and that it can be overridden?